Dockerfile-cypress

```diff
@@ -1,15 +1,23 @@
-FROM cypress/included:3.6.0
+FROM cypress/included:3.8.1
 
 ENV APP_HOME /usr/src/app
 RUN mkdir -p $APP_HOME
 WORKDIR $APP_HOME
 
-COPY . $APP_HOME
+COPY . ./
+RUN rm package-lock.json
```

Comment:
Why do we remove the lock file - this could result in getting newer versions of packages that might be incompatible?

Reply:
to make sure that the newest version of every dependency was downloaded